tests/unit/conftest.py

```diff
@@ -30,38 +31,54 @@ def patch(request):
     return functools.partial(autopatcher, request)
 
 
+@pytest.fixture(scope="session")
+def pyramid_settings():
+    return {
+        "database_url": os.environ.get(
+            "TEST_DATABASE_URL",
+            "postgresql://postgres@localhost:5434/checkmate_test",
+        )
+    }
```

Comment:
Some re-ordering in this file to bring all the request and pyramid style fixtures together